### PR TITLE
Extension Properties

### LINKED ISSUE #32: Migrate extension elements group
* [ ] re-implement as simple list group

Not implemented right now, but important for the future: extension properties can also be inside form fields.

**as is**

image

**to be**

image

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #32 

Bildschirmfoto 2021-07-19 um 16 43 44

### CODE REVIEW

**Comment by MaxTru** [comment]
Minor: the commit message is wrong `feat(camunda-platform): add execution properties group`

File: src/provider/camunda-platform/properties/ExtensionPropertiesProps.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    const {
+      selectedElement: element
+    } = useContext(BpmnPropertiesPanelContext);
+
+    const commandStack = useService('commandStack');
+
+    const removeElement = (event) => {
+      event.stopPropagation();
+
+      let commands = [];
```
Comment (MaxTru) [inline src/provider/camunda-platform/properties/ExtensionPropertiesProps.js:85]: minor: can be `const`

**Review by MaxTru — COMMENTED** [review]
Works like a charm :+1:

**Comment by MaxTru** [comment]
Can be merged after adressing my two super minor comments

File: src/provider/camunda-platform/properties/ExtensionPropertiesProps.js (hunk 1 above)
Comment (pinussilvestrus) [reply src/provider/camunda-platform/properties/ExtensionPropertiesProps.js]: 👍

**Review by MaxTru — APPROVED** [review]

**Comment by MaxTru** [comment]
rebased so that this is auto-merged

**Pull request merged**